lib/traject/marc_extractor.rb

```diff
@@ -105,6 +105,11 @@ module Traject
   # lazily create and then re-use a MarcExtractor object with
   # particular initialization arguments.
   class MarcExtractor
+    # constant values
+    EMPTY = []
```

Comment:
and some comments explaining this is done for performance and has measurable perf advantages wouldn't hurt either, so some future coder doesn't get rid of it without measuring.